packages/relay/src/lib/services/ethService/ethCommonService/index.ts

```diff
@@ -175,6 +177,20 @@ export class CommonService implements ICommonService {
     returnLatest?: boolean,
   ): Promise<any> {
     if (!returnLatest && this.blockTagIsLatestOrPending(blockNumberOrTagOrHash)) {
+      if (this.logger.isLevelEnabled('debug')) {
```

Comment:
nit: why is it needed to query for the log level? can we just interpolate the log message? https://github.com/pinojs/pino/blob/main/docs/api.md#interpolationvalues-any

This would be just to simplify the log logic.